### PR TITLE
Universal style changes

### PR DESCRIPTION
**Pull request by jenniferthibault**
Small style tweaks overall including:
 - font sizes in committee tabs
 - reducing letter spacing in tabs
 - aligning check/radio buttons with text

### CODE REVIEW

File: scss/elements/_lists.scss [hunk 1]
```diff
@@ -19,7 +19,9 @@
 ul,
 ol {
   list-style-type: none;
-  margin: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: .25rem;
```
Comment (noahmanger) [inline scss/elements/_lists.scss]: Do you just want this to apply to bulletted lists? we often use `ul` for things that we don't want to look like lists, so if this is for bullet's it should go under `list--bulleted` in `list-styles.scss`.

Comment (jenniferthibault) [inline scss/elements/_lists.scss]: Nope, I was looking at only radio buttons and check boxes here. Thanks for pointing this out though so I'll know where to look in the future!

**Comment by jmcarp** [comment]
Is this ready for merge?

**Comment by noahmanger** [comment]
I want to change the list margin stuff, but then yeah.

**Pull request merged**